bin/external_lib_deps.ml

```diff
@@ -46,18 +46,31 @@ let all_lib_deps ~request =
   |> Context_name.Map.map
        ~f:(Path.Source.Map.of_list_reduce ~f:Lib_deps_info.merge)
 
+let populate_opam_command ?switch_name packages =
```

Comment:
Apologies for the nitpicking, but I would rename this to `opam_install_command` instead. "Populate" is a bit confusing IMHO.

Reply:
Done!